src/main/webapp/app/admin/standardized-competencies/import/admin-import-standardized-competencies.component.ts

```diff
@@ -166,6 +167,7 @@ export class AdminImportStandardizedCompetenciesComponent {
                 this.importCount.knowledgeAreas -= 1;
                 this.dataSource.data = this.importData.knowledgeAreas.map((knowledgeArea) => convertToKnowledgeAreaForTree(knowledgeArea));
             }
+            // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
We can adapt the no-unused-vars rule, so errors can be unused but other variables still have to:
https://eslint.org/docs/latest/rules/no-unused-vars#caughterrors

Reply:
good idea, can we define this in the global eslint config?

Reply:
found it and applied it